roles/letsencrypt/meta/main.yml

```diff
@@ -0,0 +1,21 @@
+dependencies:
+  - role: apache
+    ssl: no
+
+    apache_mods_enabled:
+      - rewrite.load
+
+    apache_vhosts:
```

Comment:
If we have the letsencrypt vhost on a separate port, does that port become enforced in the resulting certificate?